File: LeetCodeJava/src/solution/threeSum/CheckAnswer.java
package solution.threeSum;

public class CheckAnswer {

    public static void main(String[] args) {
        Solution s = new Solution();
        int[][] input = {
                {-1, 0, 1, 2, -1, -4},
                {0, 0, 0, 0, 0, 0, 0, 0, 0},
                {-1, -1, -1, -1, 0, 1, 1, 1, 1},
                {1, 1, 1}
        };
        for (int[] i : input) {
            System.out.println(s.threeSum(i));
        }

    }

}

File: LeetCodePy/src/questions/three_sum/CheckAnswer.py
'''
Created on 2020年9月25日

@author: zwieb
'''
from questions.three_sum.Solution import Solution

s = Solution()

input_list = [
    [-1, 0, 1, 2, -1, -4],
    [0, 0, 0, 0, 0, 0, 0, 0, 0],
    [-1, -1, -1, -1, 0, 1, 1, 1, 1],
    [1, 1, 1],
]

for input in input_list:
    print(s.threeSum(input))

